frontend/src/Routing.js

```diff
@@ -8,11 +8,12 @@ import TableListofPartners from './partners/TableListofPartners'
 import CreateWorkSheet from './worksheets/CreateWorkSheet'
 import UpdateWorkSheet from './worksheets/UpdateWorkSheet'
 import AdminRoute from './AdminRoute'
+import Partner from './partners/Partner'
 
 function Routing() {
   return (
-    <div className="col-12">
-      <Switch>
+    <Switch>
+      <div className="col-12">
```

Comment:
Ez miért jobb így? Szerintem a Switch nem is szereti, ha nem Route-ok a gyerekei.